app/src/main/java/com/xmartlabs/daydreaming/ui/DashboardFragment.java

```diff
@@ -183,6 +186,20 @@ private boolean closeDrawer(@NonNull MenuItem menuItem) {
     return true;
   }
 
+  private void showRateAppDialog() {
+    new AlertDialog.Builder(getContext())
+        .setTitle(R.string.rate)
+        .setMessage(R.string.rate_app_now)
+        .setPositiveButton(R.string.rate_button, (dialog, which) -> openURL())
+        .setNegativeButton(R.string.close, (dialog, which) -> {})
+        .show();
+  }
+
+  private void openURL() {
```

Comment:
Yep, I can rename it to `openRateAppURL` or `openGooglePlayURL` because in the future it will took you to the google play url in order to rate the app.